library/x509_crt.c

```diff
@@ -818,8 +818,7 @@ static int x509_get_certificate_policies( unsigned char **p,
         {
             /*
              * Set the parsing return code but continue parsing, in case this
-             * extension is critical and MBEDTLS_X509_ALLOW_UNSUPPORTED_CRITICAL_EXTENSION
-             * is configured.
```

Comment:
If `MBEDTLS_X509_ALLOW_UNSUPPORTED_CRITICAL_EXTENSION` can no longer be configured, surely we can we just return here, rather than keep parsing?

Reply:
What if there's a callback that recognizes a policy that the library doesn't support? Doesn't this function need to keep going then?

Reply:
Ah okay, that makes sense - I was going purely off the semantics of the comment, so I didn't consider the actual code.